torch/utils/viz/MemoryViz.js

```diff
@@ -762,7 +762,12 @@ function frameFilter({name, filename}) {
 
 function format_frames(frames) {
   if (frames.length === 0) {
-    return `<block was allocated before _record_history was enabled>`;
+    return (
+      `<block has no frames, potential causes:\n` +
+      `1) block was allocated before _record_memory_history was enabled,\n` +
+      `2) context or stacks passed to _record_memory_history does not include this block,\n` +
+      `3) backward events allocated with C++ stack and will not show if stacks = python>`
```

Comment:
1 is actionable, 3 can be rephrased to be actionable. Can 2 be reworded to describe how to change _record_memory_history in order to get information?

```
3) This event occurred during backward, which has no python frames, and memory history did not include C++ frames. Use stacks='all' to record both C++ and python frames.
```

Reply:
Sounds good, I've updated the string, and matched similar style (capitalization, periods, well-formed sentences).